controllers/providers/metrics/prometheus.go

```diff
@@ -119,42 +140,57 @@ func (m *PrometheusMetrics) Unregister() {
 
 // init instantiates particular metrics
 func (m *PrometheusMetrics) init() {
-	m.metrics.HealthyRecords = prometheus.NewGaugeVec(
+	m.metrics.K8gbGslbHealthyRecords = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
-			Namespace: m.config.K8gbNamespace,
-			Subsystem: gslbSubsystem,
-			Name:      "healthy_records",
-			Help:      "Number of healthy records observed by K8GB.",
+			Name: "k8gb_gslb_healthy_records",
+			Help: "Number of healthy records observed by K8GB.",
 		},
 		[]string{"namespace", "name"},
 	)
 
-	m.metrics.IngressHostsPerStatus = prometheus.NewGaugeVec(
+	m.metrics.K8gbGslbStatusPerIngressHosts = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
-			Namespace: m.config.K8gbNamespace,
-			Subsystem: gslbSubsystem,
-			Name:      "ingress_hosts_per_status",
-			Help:      "Number of managed hosts observed by K8GB.",
+			Name: "k8gb_gslb_status_per_ingress_hosts",
+			Help: "Number of managed hosts observed by K8GB.",
 		},
 		[]string{"namespace", "name", "status"},
 	)
 
-	m.metrics.ZoneUpdateTotal = prometheus.NewCounter(
+	m.metrics.K8gbGslbErrorTotal = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
-			Namespace: m.config.K8gbNamespace,
-			Subsystem: gslbSubsystem,
-			Name:      "delegated_zone_update",
-			Help:      "Number of delegated zone updates. Is hit for create as well",
+			Name: "k8gb_gslb_error_total",
+			Help: "Number of errors",
 		},
+		[]string{"namespace", "name"},
 	)
 
-	m.metrics.ReconciliationTotal = prometheus.NewCounter(
+	m.metrics.K8gbGslbReconciliationTotal = prometheus.NewCounter(
 		prometheus.CounterOpts{
-			Namespace: m.config.K8gbNamespace,
-			Subsystem: gslbSubsystem,
-			Name:      "reconciliation_total",
-			Help:      "Number of successful reconciliation loops.",
+			Name: "k8gb_gslb_reconciliation_total",
```

Comment:
Consider making counter name plural:
```suggestion
			Name: "k8gb_gslb_reconciliations_total",
```
or
```suggestion
			Name: "k8gb_gslb_reconciliation_loops_total",
```

Reply:
Thanks @sommaritan, true. If you don't mind. I'll fix plurals it in the next PR, where I refactored everything into constants.

Reply:
@kuritka sure, no problem. There are few other comments regarding metric descriptions from my side, but I can leave them for the next PR.